crates/ibc/src/core/ics02_client/events.rs

```diff
@@ -45,12 +45,9 @@ struct ClientTypeAttribute {
     client_type: ClientType,
 }
 
-impl From<ClientTypeAttribute> for Tag {
+impl From<ClientTypeAttribute> for abci::EventAttribute {
     fn from(attr: ClientTypeAttribute) -> Self {
-        Tag {
-            key: CLIENT_TYPE_ATTRIBUTE_KEY.parse().unwrap(),
-            value: attr.client_type.to_string().parse().unwrap(),
-        }
+        (CLIENT_TYPE_ATTRIBUTE_KEY, attr.client_type.as_str()).into()
     }
 }
```

Comment:
Note that the `EventAttribute` domain type also has an `index` field, which is set to false by the `From` conversion used here.

Reply:
Hmm, it may have been too early to switch to `tendermint::abci::Event` before https://github.com/informalsystems/tendermint-rs/pull/1204 is released. There is still `tendermint_rpc::abci::Event` confusingly, but it's what hermes is using.

Reply:
Upon further examination, the ABCI event infrastructure in Hermes now duplicates this code and is not dependent on `ibc`. So we can go on with this change.